### PR TITLE
[v2.0.3] Fixed issue #2998, Retain Sites page when user clicks on load more icon and adds a site

### PR DESCRIPTION
**Pull request by monica-BTC**
Fixed issue #2998

### CODE REVIEW

File: participant-manager-datastore/participant-manager-service/src/main/java/com/google/cloud/healthcare/fdamystudies/beans/SiteResponse.java [hunk 1]
```diff
@@ -20,8 +20,11 @@
 
   private String siteId;
 
-  public SiteResponse(String siteId, MessageCode messageCode) {
+  private String siteName;
```
Comment (moschetti) [inline participant-manager-datastore/participant-manager-service/src/main/java/com/google/cloud/healthcare/fdamystudies/beans/SiteResponse.java:23]: Please update file copyright to `2020-2021`

File: participant-manager-datastore/participant-manager-service/src/main/java/com/google/cloud/healthcare/fdamystudies/mapper/SiteMapper.java [hunk 2]
```diff
@@ -22,6 +22,7 @@ private SiteMapper() {}
   public static SiteResponse toSiteResponse(SiteEntity site) {
     SiteResponse response = new SiteResponse();
     response.setSiteId(site.getId());
+    response.setSiteName(site.getLocation().getName());
```
Comment (moschetti) [inline participant-manager-datastore/participant-manager-service/src/main/java/com/google/cloud/healthcare/fdamystudies/mapper/SiteMapper.java:25]: Please update file copyright to `2020-2021`

File: participant-manager-datastore/participant-manager-service/src/main/java/com/google/cloud/healthcare/fdamystudies/beans/SiteResponse.java (hunk 1 above)
Comment (monica-BTC) [reply participant-manager-datastore/participant-manager-service/src/main/java/com/google/cloud/healthcare/fdamystudies/beans/SiteResponse.java:23]: Updated file to `2020-2021`.

File: participant-manager-datastore/participant-manager-service/src/main/java/com/google/cloud/healthcare/fdamystudies/mapper/SiteMapper.java (hunk 2 above)
Comment (monica-BTC) [reply participant-manager-datastore/participant-manager-service/src/main/java/com/google/cloud/healthcare/fdamystudies/mapper/SiteMapper.java:25]: Updated file to `2020-2021`.

**Review by moschetti — APPROVED** [review]

**Pull request merged**